lib/generators/arclight/templates/catalog_controller.rb

```diff
@@ -97,6 +97,7 @@ class CatalogController < ApplicationController
     config.add_facet_field 'access_subjects_ssim', label: 'Subject', limit: 10
     config.add_facet_field 'component_level_isim', show: false
     config.add_facet_field 'parent_ssim', show: false
+    config.add_facet_field 'parent_ssi', show: false
```

Comment:
What's the difference between (the content indexed into) `parent_ssim` and `parent_ssi`? Can we consolidate, or at least leave a comment about what they're both used for (or maybe the `ssim` one is a typo)?

Reply:
Thanks @cbeer -- that's a good question, and it's reasonable to assume that could be a typo. `parent_ssim` is an array of the all of the component's ancestor node IDs, from the top-level collection down to its immediate parent. `parent_ssi` is the just final value of the array: the direct parent's ID.

To me, the best way to disambiguate is ultimately to replace the words `parent` or `parents` with `ancestors` throughout the app in all cases except when we truly mean just the immediate parent. That'd mean changes to content/context navigation JS, Solr configs, traject rules, etc. so it'd be a heavier lift that might be better timed with other breaking changes in the future.

I'd be glad to amend this PR with some inline comments to clarify.

Reply:
Thanks, that helps my understanding. I've 👍'd this and will happily merge it after giving others (who may have more context for arclight...) a chance to chime in.